fjord/feedback/jinja2/feedback/android_feedback.html

```diff
@@ -0,0 +1,192 @@
+                </div>
+
+                <noscript>
+                  <p class="error">
+                    {% trans %}
+                      JavaScript is required to leave feedback. Please enable JavaScript in your browser and refresh this page.
+                    {% endtrans %}
+                  </p>
+                </noscript>
+
+                <div id="happy-button" class="link-box sentiment-button" tabindex=0>
+                  <a>
+                    {% trans %}
+                      I love it
+                    {% endtrans %}
+                  </a>
+                </div><!-- #happy-button -->
+
+                <div id="sad-button" class="link-box sentiment-button" tabindex=0>
+                  <a>
+                    {% trans %}
+                      I ran into some problems
+                    {% endtrans %}
+                  </a>
+                </div><!-- #sad-button -->
+
+              </section>
+            </div><!-- #intro.card -->
+
+            <div class="card inactive" id="moreinfo" data-focus="#description" data-back-id="intro">
+              <section>
+                <div class="happy">
+                  <h1>
+                    {% trans %}
+                      That's great to hear!
+                    {% endtrans %}
+                  </h1>
+
+                  <div class="message-box">
+                    <div class="message">
+                      {% trans %}
+                        Want to share the love by giving us a 5-star rating on
+                        Google Play?
+                      {% endtrans %}
+                    </div><!-- .message -->
+                    <div>
+                      {% trans %}
+                        It takes less than a minute and feels great.
+                      {% endtrans %}
+                    </div>
+                    <a class="stars">
+                      <svg>
+                        <defs>
+                          <symbol viewBox="0 0 100 100" id="star">
+                            <polygon points="48 75 18.6107374 90.4508497 24.2235871 57.7254249 0.447174185 34.5491503 33.3053687 29.7745751 48 0 62.6946313 29.7745751 95.5528258 34.5491503 71.7764129 57.7254249 77.3892626 90.4508497" class="star" xmlns="http://www.w3.org/2000/svg" fill="#0092db"></polygon>
+                          </symbol>
+                        </defs>
+                        <use width="25" height="25" xlink:href="#star" x="0" y="0"></use>
+                        <use width="25" height="25" xlink:href="#star" x="25" y="0"></use>
+                        <use width="25" height="25" xlink:href="#star" x="50" y="0"></use>
+                        <use width="25" height="25" xlink:href="#star" x="75" y="0"></use>
+                        <use width="25" height="25" xlink:href="#star" x="100" y="0"></use>
+                      </svg>
+                      {% trans %}
+                        Yes, go to Google Play
+                      {% endtrans %}
```

Comment:
Javascript now includes logic to switch this link based on a url parameter we're getting from the in-product feedback page: https://github.com/mozilla/fjord/pull/723/files#diff-aacf79c42cfad2e29069461b3eb426a8R9

If parameter is present, URL is an app intent; otherwise it's http.